### DISCUSSION TITLE
Sharing custom routes

### DISCUSSION THREAD

**Discussion by jackthebean** [osmandapp/OsmAnd#21411] [Maps Related]
I would like to use UK driving test routes for practice with my children.   I cannot find any GPX files to download that someone else has already created and so am happy to create my own if necessary as I have found the text versions.   However is there a place that people might upload such GPX files so that I can check that they have not already been created and, if not, that I can upload my GPX files to so that someone else can use them?
Thanks for your advice,

**Comment by DmitryAlexei** [comment]
Some driving routes you can find on the OsmAnd site (see link https://osmand.net/map/travel/#10/53.6202/-2.7988 ). However, they are not specified as test routes, so you need to find suitable for you.

**Reply by jackthebean** [reply]
Thanks a lot.  Have had a look, unable to find any and so have created 2 (one on phone prior) and uploaded both.   All fine except sometimes it tells me that I cannot create tracks etc as don't have Osmand Pro subscription though think I have and other times it lets me, shows me the ones I have created etc.  A bit odd in actual usage.
